packages/jest-environment-jsdom/src/index.js

```diff
@@ -4,15 +4,27 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
+ * @flow
  */
 'use strict';
 
 const FakeTimers = require('jest-util').FakeTimers;
 const installCommonGlobals = require('jest-util').installCommonGlobals;
 
+import type {Global} from 'types/Global';
+
+type Config = {
```

Comment:
there is a Config type in types/Config. It might miss testURL, so please just add it.